docs/keto/sdk/go.md

```diff
@@ -3,9 +3,7 @@ id: go
 title: Go
 ---
 
-import CodeBlock from '@theme/CodeBlock'
-import { useLatestRelease } from '@site/src/hooks'
-
+import CodeBlock from '@theme/CodeBlock'; import { useLatestRelease } from '@site/src/hooks'
```

Comment:
If we rename this file to `go.mdx`, then Prettier doesn't merge these two lines. I think given that this file contains Gatsby stuff it should have that extension anyways, no?

Reply:
👍

Reply:
Maybe this PR is a good opportunity to make sure all files use .mdx extensions instead of .md? 🤔

Reply:
That's a separate change from enabling Prettier, so I think it deserves a separate PR.

And given how massive this PR is, I would prefer to get it merged with the fewest changes possible. What I want to avoid is that anybody has to review 329 files again. 

I'm happy to submit a new PR that fixes file extensions right after this one goes in.

Reply:
Fair point. This is something either of us can handle when this one is merged.